src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -1,14 +1,15 @@
 @import 'src/styles/common';
-@import 'constants';
+@import '../../constants';
```

Comment:
maybe change to `src/content/app/entity-viewer/gene-view/styles/constants`? This way, files can be more easily moved around